webpack/webpack.prod.js

```diff
@@ -0,0 +1,15 @@
+const webpack = require('webpack');
+const CompressionPlugin = require("compression-webpack-plugin");
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+module.exports = {
+    mode: 'production',
+    devtool: 'source-map',
+    plugins: [
+        new CompressionPlugin({
+            filename: "[path][name].jgz",
+            test: /\.(ts|js)x?$/,
+        }),
+        new BundleAnalyzerPlugin(),
```

Comment:
As a suggestion add the following - so that it doesn't startup a server for the report every time a production bundle is generated and also disable it from automatically opening the report in the browser. 
```
 new BundleAnalyzerPlugin({
   analyzerMode: 'static',
   openAnalyzer: false
}),
```